data/moves.js

```diff
@@ -9824,6 +9824,7 @@ let BattleMovedex = {
 			}
 			// Instructed Fake Out etc. should fail
 			target.activeTurns++;
+			target.totalActiveTurns++;
```

Comment:
This is a horrible hack I think?

Reply:
Why? if it counts towards increasing the active Pokemon's turn count, it should increase the overall turn count.

Reply:
Yes, and increasing the active turns is a horrible hack

Reply:
Piling hacks upon hacks is a bad idea, I think.

Reply:
I'll remove it for now then.